Libraries/LibWeb/CSS/StyleValues/CalculatedStyleValue.cpp

```diff
@@ -2927,7 +2927,9 @@ NonnullRefPtr<CalculationNode const> simplify_a_calculation_tree(CalculationNode
     if (root->is_math_function_node()) {
         if (auto maybe_simplified = root->run_operation_if_possible(context, resolution_context); maybe_simplified.has_value()) {
             // NOTE: If this returns nullptr, that's a logic error in the code, so it's fine to assert that it's nonnull.
-            return make_calculation_node(maybe_simplified.release_value(), context).release_nonnull();
```

Comment:
It seems like the comment above isn't valid anymore.

Reply:
Touché!